README.md

```diff
@@ -1 +1,193 @@
-# todo-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <h3><b>To Do list: list structure</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 To Do list: list structure <a name="about-project"></a>
+
+**In this project, I will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.**
+
+
+
+![screenshot](images/awesome_books_full_website.png)
+
```

Comment:
- [ ] Please kindly fix this broken link. **Hint: I think there is no images directory in your root folder.** 👍